bftengine/src/bftengine/messages/ClientRequestMsg.cpp

```diff
@@ -102,11 +102,19 @@ void ClientRequestMsg::validateImp(const ReplicasInfo& repInfo) const {
   const auto* header = msgBody();
   PrincipalId clientId = header->idOfClientProxy;
   if ((header->flags & RECONFIG_FLAG) == 0) ConcordAssert(this->senderId() != repInfo.myId());
+
+  const auto msgSize = size();
+  std::stringstream msg;
+  // Check message size is greater than muinimum header size
```

Comment:
sorry, i missed it earlier:
the spelling of minimum should be corrected

Reply:
Ok